### DISCUSSION TITLE
Allow changing of version order

### DISCUSSION THREAD

**Discussion by hassan-sd** [civitai/civitai#57] [Ideas]
The versions are sorted by how they are uploaded and not by numerical identifier.
If we have 5 models created and batch upload them all in one go, it will state the "recent" version is just the recent one uploaded.

Allow us to edit a model and customize which model is most recent and tied to the Download Latest option here:

![image](https://user-images.githubusercontent.com/119671806/205287129-b57ec117-a27b-4f03-b36a-e6ed497e05ff.png)

![image](https://user-images.githubusercontent.com/119671806/205287177-e5d9fba3-602b-4b7f-812d-0ea125b3a2b0.png)

**Comment by bkdiehl** [comment]
This should be live soon
![image](https://user-images.githubusercontent.com/11724394/205371587-02b290b3-882d-4306-ac41-2fea6da0fc62.png)
